posts/inside-rust/2020-02-11-Goverance-wg

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Governance Working Group Update: Meeting 11 February 2020"
+author: Val Grimm
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
+---
+
+Hello everyone! 
+
+The governance working group meeting of January 28th was canceled due to the Mozilla All Hands that week.
+Therefore, working group next convened on 11 February.
+The following meeting will take place on 27 February.
+
+The agenda included 
+1. Discuss project groups description RFC 
+https://github.com/rust-lang/rfcs/pull/2856
+Decision: Determine if inactive members in scope; if not in scope, confirm RFC. 
+2. Review action items from meeting of December 3 2019
+https://github.com/rust-lang/wg-governance/blob/master/minutes/2019.12.03.md#action-items
+3. Confirm adjustment of timeslot based on Doodle sent out in Discord channel
+
+## Project groups
+
+You can find the [detailed minutes] on the [wg-governance] repository, but here is a quick summary: 
+
+* Project groups description RFC confirmed complete
+* Next items taken from Dec 3 backlog are access policy (NellShamrell is working on this),
+Discussion of where to discuss the pre-RFC process, and followup about moving (community, devtools, nursery) to rust-lang
+* XAMPPRocky is already working on a compliance bot to simplify administration
+* Timeslot adjusted to 19-20 CET / 1pm-2pm EST / 10-11am PST on Zulip, with same biweekly cadence.
+Zoom meetings will be used as needed, e.g. if a big RFC needs to be pushed through.
```

Comment:
Could you move these last two lines under "Next meeting", and mention that we're switching from video to text for meetings.

Reply:
Ok, will do.